browser/ui/tabs/brave_tab_strip_model.cc

```diff
@@ -151,6 +152,25 @@ void BraveTabStripModel::SetCustomTitleForTab(
   NotifyTabChanged(tab_interface, TabChangeType::kAll);
 }
 
+void BraveTabStripModel::SetCustomEmojiFaviconForTab(
```

Comment:
This is also not the right place for this, the TabStripModel is not responsible for managing tab icons